CONTRIBUTING.md

```diff
@@ -48,4 +49,30 @@ In order for code to be merged into master it must
 The reviewers are expected to look out for the items above before approving a merge
 request.
 
-
+#### Formatting Code
+We use clang-format to automatically format the code. If you have clang-format installed
+locally, you can always execute `make format` or `cmake --build . --target format` from
+your build directory to automatically format the code.
+
+If you don't have clang-format installed locally, our "Hermes" automation can always
+format the code for you. Just create the following comment in your PR, and Hermes will
+format the code and automatically commit it:
+```
+@par-hermes format
+```
```

Comment:
This is really nice.

I wonder if we should also include directions for adding a commit hook locally if people want to do that?

Reply:
Could this be done at a later time?